src/main.py

```diff
@@ -178,6 +185,60 @@ def run_plan_bot(
             skip_tracking=skip_tracking,
         )
 
+    params = {}
+    if comments_progress:
+        newest_comment_id = comments_progress.get("newest")
+        oldest_comment_id = comments_progress.get("oldest")
+
+        if newest_comment_id:
+            params["before"] = latest_comment_id
+
+    # Get comments newer than latest
+    current_newest_comment_id = None
+    for comment in subreddit.comments(params=params):
```

Comment:
How are these results sorted? Is it oldest first or newest first? Seems oldest first would be desired (and in which case would we even need the oldest_comment logic?)

If it's newest first by default, can we pass `{ "sort": "old" }` or `{"sort":"new"}` to make it oldest first?

Reply:
My tests indicate that its newest first. I don't see a `sort` parameter [here](https://www.reddit.com/dev/api/#listings) but I didn't try to do that. Have you experienced that parameter having any effect?